### PR TITLE
Upgrade to Fluture and Node version 12

### PR DESCRIPTION
**Pull request by Avaq**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  input: 'index.js',
+  external: ['fluture/index.js', 'callgebra/index.js'],
+  output: {
+    format: 'umd',
+    name: 'flutureHooks',
+    file: 'index.cjs',
+    interop: false,
+    paths: {
+      'fluture/index.js': 'fluture',
+      'callgebra/index.js': 'callgebra',
```
Comment (Avaq) [inline rollup.config.js:11]: This is what I meant with choice iii in https://github.com/fluture-js/Fluture/pull/383#issue-324290164

File: package.json [hunk 2]
```diff
@@ -23,13 +24,15 @@
   "author": "Aldwin Vlasblom <aldwin@avaq.it> (https://github.com/Avaq)",
   "license": "MIT",
   "dependencies": {
-    "callgebra": "^1.0.0"
+    "callgebra": "github:fluture-js/callgebra#avaq/maintenance"
```
Comment (Avaq) [inline package.json]: Depends on the release of callgebra

File: package.json [hunk 3]
```diff
@@ -23,13 +24,15 @@
   "author": "Aldwin Vlasblom <aldwin@avaq.it> (https://github.com/Avaq)",
   "license": "MIT",
   "dependencies": {
-    "callgebra": "^1.0.0"
+    "callgebra": "^2.0.0"
   },
   "peerDependencies": {
-    "fluture": ">=11.0.0 <12.0.0"
+    "fluture": ">=12.0.0-beta.4 <13.0.0"
```
Comment (dicearr) [inline package.json:30]: Do we want to support `12.0.0-beta.4`, `12.0.0-beta.5` and `12.0.0-beta.6`?

Comment (Avaq) [reply package.json:30]: Yeah, they work, so why not.

**Review by dicearr — APPROVED** [review]

**Pull request merged**